WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
Maybe something briefer like `advancing_cutoff` or `proceed_cutoff`?
@larspetrus do you have any opinion on that?